population_from_wikipedia.js

```diff
@@ -0,0 +1,243 @@
+var population = {
+"China": 1401855920,
+"India": 1360053900,
+"US": 329488021,
+"Indonesia": 266911900,
+"Pakistan": 219016520,
+"Brazil": 211283757,
+"Nigeria": 206139587,
+"Bangladesh": 168304408,
+"Russia": 146745098,
+"Mexico": 126577691,
+"Japan": 126010000,
+"Philippines": 108433682,
+"Egypt": 100150004,
+"Ethiopia": 98665000,
+"Vietnam": 96208984,
+"DR Congo": 89561404,
+"Iran": 83298325,
+"Turkey": 83154997,
+"Germany": 83149300,
+"France": 67069000,
+"Thailand": 66483955,
+"United Kingdom": 66435600,
+"Italy": 60243406,
+"South Africa": 58775022,
+"Tanzania": 55890747,
+"Myanmar": 54339766,
+"Korea, South": 51780579,
+"Colombia": 49395678,
+"Kenya": 47564296,
+"Spain": 47100396,
+"Argentina": 44938712,
+"Algeria": 43000000,
+"Sudan": 42361520,
+"Ukraine": 41902416,
+"Uganda": 40299300,
+"Iraq": 39127900,
+"Poland": 38386000,
+"Canada": 37965057,
+"Morocco": 35845131,
+"Saudi Arabia": 34218169,
+"Uzbekistan": 34081430,
+"Malaysia": 32725760,
+"Afghanistan": 32225560,
+"Venezuela": 32219521,
+"Peru": 32131400,
+"Angola": 31127674,
+"Ghana": 30280811,
+"Mozambique": 30066648,
+"Yemen": 29825968,
+"Nepal": 29609623,
+"Cameroon": 26545864,
+"Ivory Coast": 25823071,
+"Madagascar": 25680342,
+"Australia": 25652995,
+"North Korea": 25450000,
+"Taiwan": 23604265,
+"Niger": 22314743,
+"Sri Lanka": 21803000,
+"Burkina Faso": 20870060,
+"Mali": 19973000,
+"Romania": 19405156,
+"Malawi": 19129952,
+"Chile": 19107216,
+"Kazakhstan": 18667080,
+"Syria": 17500657,
+"Ecuador": 17448612,
+"Netherlands": 17446831,
+"Zambia": 17381168,
+"Guatemala": 16604026,
+"Senegal": 16209125,
+"Somalia": 15893219,
+"Chad": 15692969,
+"Cambodia": 15288489,
+"Zimbabwe": 15159624,
+"South Sudan": 12778250,
+"Rwanda": 12374397,
+"Guinea": 12218357,
+"Benin": 11733059,
+"Tunisia": 11722038,
+"Haiti": 11577779,
+"Belgium": 11524454,
+"Bolivia": 11469896,
+"Cuba": 11209628,
+"Burundi": 10953317,
+"Greece": 10724599,
+"Czech Republic": 10693939,
+"Jordan": 10640708,
+"Dominican Republic": 10358320,
+"Sweden": 10333456,
+"Portugal": 10276617,
+"Azerbaijan": 10067108,
+"United Arab Emirates": 9890400,
+"Hungary": 9772756,
+"Belarus": 9413446,
+"Israel": 9174600,
+"Honduras": 9158345,
+"Tajikistan": 9127000,
+"Papua New Guinea": 8935000,
+"Austria": 8902600,
+"Switzerland": 8586550,
+"Sierra Leone": 7901454,
+"Togo": 7538000,
+"Hong Kong": 7500700,
+"Paraguay": 7152703,
+"Laos": 7123205,
+"Bulgaria": 7000039,
+"Serbia": 6963764,
+"Libya": 6871287,
+"Lebanon": 6825442,
+"Kyrgyzstan": 6523500,
+"El Salvador": 6486201,
+"Nicaragua": 6460411,
+"Turkmenistan": 6031187,
+"Denmark": 5822763,
+"Singapore": 5703600,
+"Finland": 5527573,
+"Congo": 5518092,
```

Comment:
`Congo (Kinshasa)` or `Congo (Brazzaville)`?

Reply:
Yeah, they were just crawled from Wiki, I went through some obvious differences, but some are hard...

Reply:
Adjusted everything to match JHU data.